package.json

```diff
@@ -62,7 +62,7 @@
     ]
   },
   "peerDependencies": {
-    "esri-loader": "^2.10",
+    "esri-loader": "^2.14",
```

Comment:
Unless we are forcing people to 4.15, we should **not** bump `peerDependencies` this should be left as loose as possible.